src/components/plots/plot.ts

```diff
@@ -540,5 +563,44 @@ module Plottable {
       return !(pixelPoint.x < 0 || pixelPoint.y < 0 ||
         pixelPoint.x > this.width() || pixelPoint.y > this.height());
     }
+
+    protected _updateExtentsForProperty(property: string) {
+      var accScaleBinding = this._propertyBindings.get(property);
+      if (accScaleBinding.accessor == null) { return; }
```

Comment:
Is this a worthwhile safety check? It seems like in this case, instead of returning immediately, the method should set the extents to `[]` -- otherwise `_propertyExtents` will contain outdated information.

Also, I'm surprised TSLint didn't yell about the single-line `if`.